clients/vscode-hlasmplugin/schema/asm_options.json

```diff
@@ -1,56 +1,148 @@
 {
-    "type": "object",
-    "description": "List of assembler options",
-    "properties": {
-        "GOFF": {
-            "type": "boolean",
-            "description": "Produces Generalized Object File format dataset."
-        },
-        "OPTABLE": {
-            "type": "string",
-            "description": "Specifies the instruction set to use.",
-            "enum": [
-                "UNI",
-                "DOS",
-                "370",
-                "XA",
-                "ESA",
-                "ZOP",
-                "ZS1",
-                "YOP",
-                "ZS2",
-                "Z9",
-                "ZS3",
-                "Z10",
-                "ZS4",
-                "Z11",
-                "ZS5",
-                "Z12",
-                "ZS6",
-                "Z13",
-                "ZS7",
-                "Z14",
-                "ZS8",
-                "Z15",
-                "ZS9"
-            ]
+  "type": "object",

```

Comment:
reformat?

Reply:
Yeah, it is formatted. Maybe we have different formatting settings?